shared/test/test_channels.rb

```diff
@@ -23,19 +23,36 @@ def test_create_channel
   end
 
   def test_update_channel
+    start = Time.now - 1
     post '/v3/channels', {abc: 123}.to_json, 'CONTENT_TYPE' => 'application/json;charset=utf-8'
     channel_id = last_response.location.split('/').last
 
     get "/v3/channels/#{channel_id}"
     assert last_response.ok?
-    assert_equal 123, JSON.parse(last_response.body)['abc']
+    result = JSON.parse(last_response.body)
+    assert_equal 123, result['abc']
 
+    # Check timestamps.
+    created = result['createdAt']
+    assert_equal created, result['updatedAt']
+    assert (start..Time.now).cover? Time.parse(created)
+
+    sleep 1
+
+    # Update.
+    start = Time.now - 1
     post "/v3/channels/#{channel_id}", {abc: 456}.to_json, 'CONTENT_TYPE' => 'application/json;charset=utf-8'
     assert last_response.successful?
 
     get "/v3/channels/#{channel_id}"
     assert last_response.ok?
-    assert_equal 456, JSON.parse(last_response.body)['abc']
+    result = JSON.parse(last_response.body)
+    assert_equal 456, result['abc']
+
+    # Check timestamps.
+    assert_equal created, result['createdAt']
```

Comment:
also check that `value.created_at` is correct? It appears from the code that it might not be